Makefile

```
@@ -36,22 +39,37 @@ ifdef CUSTOM_CA
 	cp -f ${CUSTOM_CA} docker/conf/certificates/
 endif
 
+# Builds the vagrant box and the example/standalone
 up-standalone: update-box custom_ca
 ifeq ($(GITHUB_ACTIONS),true) # Always set to true when GitHub Actions is running the workflow. You can use this variable to differentiate when tests are being run locally or by GitHub Actions.
 	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} ANSIBLE_ARGS='--extra-vars "\"ci_test=true mode=standalone\""' vagrant up --provision
 else
 	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} CUSTOM_CA=${CUSTOM_CA} ANSIBLE_ARGS='--extra-vars "\"mode=standalone\""' vagrant up --provision
 endif
 
-up: update-box custom_ca
+# Builds the vagrant box and the example/standalone_git
+# Remember the parameters for GIT integration explained in /example/standalone_git/README.md
+# Example: make up repo=<GitHub-repository> branch=<branch to checkout and track> user=<GitHub username> token=<personal token from GitHub>
+up: check-params update-box custom_ca
 ifeq ($(GITHUB_ACTIONS),true) # Always set to true when GitHub Actions is running the workflow. You can use this variable to differentiate when tests are being run locally or by GitHub Actions.
-	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} ANSIBLE_ARGS='--extra-vars "\"ci_test=true mode=standalone_git\""' vagrant up --provision
+	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} ANSIBLE_ARGS='--extra-vars "\"ci_test=true mode=standalone_git repo=$(repo) branch=${branch} user=${user} token=${token}\""' vagrant up --provision
 else
-	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} CUSTOM_CA=${CUSTOM_CA} ANSIBLE_ARGS='--extra-vars "\"mode=standalone_git\""' vagrant up --provision
+	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} CUSTOM_CA=${CUSTOM_CA} ANSIBLE_ARGS='--extra-vars "\"mode=standalone_git repo=$(repo) branch=${branch} user=${user} token=${token}\""' vagrant up --provision
 endif
 
+# Checks that all parameters are set.
+check-params:
+	@[ "${repo}" ] || ( echo ">> The parameter repo is not defined. repo=<GitHub-repository, use HTTPS>" )
+	@[ "${branch}" ] || ( echo ">> The parameter branch is not defined. branch=<branch to checkout and track>")
+	@[ "${user}" ] || (  echo ">> The parameter user is not defined. user=<GitHub username>")
+	@[ "${token}" ] || ( echo ">> The parameter token is not defined. token=<personal token from GitHub" )
+	@[ "${repo}" ] && [ "${branch}" ] && [ "${user}" ] && [ "${token}" ]|| (echo "See README.md for more details and example: https://github.com/hannemariavister/terraform-nomad-nifiregistry/blob/master/example/standalone_git/README.md" ; exit 1 )
```

```suggestion
	@[ "${repo}" ] && [ "${branch}" ] && [ "${user}" ] && [ "${token}" ]|| (echo "See README.md for more details and example: https://github.com/hannemariavister/terraform-nomad-nifi/blob/master/example/standalone_git/README.md" ; exit 1 )
```
I suggest a change to https://github.com/hannemariavister/terraform-nomad-nifi/blob/master/example/standalone_git/README.md (`terraform-nomad-nifi`, not `terraform-nomad-nifiregistry`)

---

I agree, i was not sure about it